### ISSUE TITLE
bk distfiles fail to fetch

### ISSUE THREAD

**Issue by maxice8** [voidlinux/void-packages#8836]
### System

* xuname: Void 4.13.9_1 x86_64 GenuineIntel uptodate rrmFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
* architecture:  x86_64
* libc:  glibc
* package:  bk-7.3ce.1_2

### Expected behavior

bk distfile to be downloaded 

### Actual behavior

``` 
=> bk-7.3ce.1_2: running do-fetch hook: 00-distfiles ...
=> bk-7.3ce.1_2: fetching distfile 'bk-7.3.1ce.src.tar.gz'...
looking up www.bitkeeper.org
connecting to www.bitkeeper.org:443
https://www.bitkeeper.org/downloads/7.3.1ce/bk-7.3.1ce.src.tar.gz: Connection refused
=> ERROR: bk-7.3ce.1_2: failed to fetch bk-7.3.1ce.src.tar.gz.
```

### Steps to reproduce the behavior

1. Clone xbps-src
2. try to fetch distfiles for bk package

**Comment by chneukirchen** [comment]
bk 7.3.2 is out. Will update.

**Comment by chneukirchen** [comment]
Fixed in 8708434614
